Simplenote.xcodeproj/project.pbxproj

```diff
@@ -391,6 +391,17 @@
 		B5F3FFFF23F44679007A7C59 /* SPSortBar.swift in Sources */ = {isa = PBXBuildFile; fileRef = B5F3FFFE23F44679007A7C59 /* SPSortBar.swift */; };
 		BA5C1C0725BF9D6C006E3820 /* SPDragBar.swift in Sources */ = {isa = PBXBuildFile; fileRef = BA5C1C0625BF9D6C006E3820 /* SPDragBar.swift */; };
 		D435D38FCA5863E8447D5C2C /* Pods_Automattic_SimplenoteShare.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 183BAB896957F07FDCAB6DF5 /* Pods_Automattic_SimplenoteShare.framework */; };
+		D864B16925CAE47C00F9B73E /* Generic.swift in Sources */ = {isa = PBXBuildFile; fileRef = D864B16825CAE47C00F9B73E /* Generic.swift */; };
+		D864B18225CAE4BE00F9B73E /* SimplenoteUITests.swift in Sources */ = {isa = PBXBuildFile; fileRef = D864B18125CAE4BE00F9B73E /* SimplenoteUITests.swift */; };
+		D864B19225CAE54300F9B73E /* AllNotesClass.swift in Sources */ = {isa = PBXBuildFile; fileRef = D864B19125CAE54300F9B73E /* AllNotesClass.swift */; };
```

Comment:
Nitpick on the file names: I find the "Class" prefix unnecessary. The main type in the file is `AllNotes` and if a reader opens it, they'll see it's a `class`.

Xcode also clearly shows that it's a class in the Open Quickly (<kbd>Shift Cmd O</kbd>) view:

![image](https://user-images.githubusercontent.com/1218433/107312499-b2377a00-6ae4-11eb-8fc8-77dbd49c7884.png)

Reply:
Renamed the files, thanks!